### DISCUSSION TITLE
What is CSS specificity?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#184] [Q&A] [answered]
In short.

**Comment by AbidHasanPiash** [comment, marked as answer]
1. in line
2. ID
3. Class
4. Outside

**Comment by Preetiraj3697** [comment]
CSS style applied by referencing external stylesheet has lowest precedence and is overridden by Internal and inline CSS.

**Comment by razuahmed75** [comment]
help me to achieve galaxy brain
